tiingo/api.py

```diff
@@ -155,9 +161,12 @@ def get_news(self, tickers=[], tags=[], sources=[], startDate=None,
             'endDate': endDate
         }
         response = self._request('GET', url, params=params)
-        return response.json()
+        if fmt=='json':
+            return response.json()
+        elif fmt=='object':
+            return json.loads(json.dumps(response.json()), object_hook=lambda d: namedtuple('X', d.keys())(*d.values()))
```

Comment:
Same comment as above for refactoring response.json() and `X`, a good name here could be "NewsArticle"